src/topics/workflows.md

```diff
@@ -439,13 +440,12 @@ steps:
 ```
 
 Here we've added a new field to the step `echo` called `scatter`. This field tells the
-runner that we'd like to scatter over this input for this particular step. Note that
+runner that we'd like to scatter over this input for this particular step. An input parameter may be listed more than once, if a parameter is listed more than once, it becomes
```

Comment:
I think shorter lines are better for others to read/write, and also to use `git` and GitHub UI diff. See “semantic line breaks” for more.

- https://sembr.org/
- https://discuss.python.org/t/semantic-line-breaks/13874/3

Reply:
Thanks! I'll make corrections to it